### PR TITLE
[Fixes #6800] Introduce picker component (and expose activity-loading, list)

### LINKED ISSUE #6800: Expose a drop-down component to extensions
### User Story

As a developer, I want a simple drop down like component to select an item from a list.
The item rendering should allow for complex view.

### Description

*Type*: Feature

### PR DESCRIPTION
**Pull request by jeluard**
fixes #6800

### Summary:

Expose picker component to extensions.

status: ready

### CODE REVIEW

File: components/src/status_im/ui/components/react.cljs [hunk 1]
```diff
@@ -142,6 +144,18 @@
 (defn list-item [component]
   (reagent/as-element component))
 
+(defn value->picker-item [{:keys [value label]}]
+  [picker-item-class {:value value :label label}])
+
+(defn picker [{:keys [style on-change selected enabled data]}]
+  (println "picker!!" selected)
```
Comment (flexsurfer) [inline components/src/status_im/ui/components/react.cljs]: leftover?

**Review by flexsurfer — APPROVED** [review]

File: components/src/status_im/ui/components/react.cljs (hunk 1 above)
Comment (jeluard) [reply components/src/status_im/ui/components/react.cljs]: Good catch! Fixed

**Review by goranjovic — APPROVED** [review]

**Comment by asemiankevich** [comment]
@jeluard build if failing all the time (

**Comment by asemiankevich** [comment]
@jeluard feel free to merge, 1 failing test is not related to this PR

**Pull request merged**